package.json

```diff
@@ -10,12 +10,12 @@
     "lint": "tslint src/**/*",
     "pretest": "npm run build",
     "start": "webpack-dev-server",
-    "tdd": "concurrently -k 'jest dist/test.js --watch' 'npm run start' 'npm run watch'",
-    "test": "jest dist/test.js",
+    "tdd": "concurrently -k 'jest --watch' 'npm run start' 'npm run watch'",
+    "test": "jest",
     "watch": "webpack -w"
   },
   "dependencies": {
-    "babydux": "^1.3.0",
+    "babydux": "^2.0.1",
```

Comment:
Upgrade removed logging to console by default, making debugging tests less noisy.